manifest/default.xml

```diff
@@ -60,6 +61,10 @@
 
   <project name="gocb" path="godeps/src/github.com/couchbase/gocb" remote="couchbase" revision="67fb8fa4c62451372108f31c93034c07d9d9f4ab"/>
 
+  <project name="gocbcore" path="godeps/src/gopkg.in/couchbase/gocbcore.v2" remote="couchbase" revision="22d72f05785ba26be033aa8b6726435b5e50a7a0"/>
+
+  <project name="jsonx" path="godeps/src/gopkg.in/couchbaselabs/jsonx.v1" remote="couchbaselabs" revision="5b7baa20429a46a5543ee259664cc86502738cad"/>
+
```

Comment:
Here is a place where it's specifically referencing `gopkg.in` [in gocb](https://github.com/couchbase/gocb/blob/b1fd6be8f3a5dd3f1b5604cf0834113e2ce07cfe/cluster_query.go#L7)

I don't know of a way to get around doing the same.

Although .. I did notice that in branch-master.xml it [doesn't point to gopkg.in](https://github.com/couchbase/manifest/blob/master/branch-master.xml#L55), so maybe this is somehow possible.  

I'm actually more confused on why [branch-master.xml](https://github.com/couchbase/manifest/blob/master/branch-master.xml) doesn't even need to reference gocbcore, which makes me think it's not even being built

@ceejatec any idea?